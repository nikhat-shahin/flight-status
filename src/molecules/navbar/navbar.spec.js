import React from 'react';
import renderer from 'react-test-renderer';
import { shallow, configure } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from "enzyme-adapter-react-16";
import { createSerializer } from 'enzyme-to-json';
import NavBar from './index';

configure({ adapter: new Adapter() });
expect.addSnapshotSerializer(createSerializer({ mode: 'deep' }));

it('renders correctly', () => {
  const component = renderer.create(
    <NavBar />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

it('should set classname for mobile UI and match snaps', () => {
 
  const component = shallow(
    <NavBar isMobile={true}  />
  );
  expect(toJson(component)).toMatchSnapshot();
});