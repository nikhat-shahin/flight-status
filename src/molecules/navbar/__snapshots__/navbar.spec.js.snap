// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <nav
    className="navbar"
  >
    <h1>
      NIKHAT SHAHIN KHAN
    </h1>
  </nav>
</div>
`;

exports[`should set classname for mobile UI and match snaps 1`] = `
<div>
  <nav
    className="navbar"
  >
    <h1>
      NIKHAT SHAHIN KHAN
    </h1>
  </nav>
</div>
`;
